index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+	<head>
```

Comment:
Check indentation. In general, we use 2 spaces.
You can configure the code editor to always do the same.